wpilibj/src/main/java/edu/wpi/first/wpilibj/motorcontrol/MotorController.java

```diff
@@ -37,6 +37,15 @@ default void setVoltage(double outputVolts) {
    */
   double get();
 
+  /**
+   * Common interface for getting the current voltage of the motor controller.
+   *
+   * @return The current controller voltage output. Value is between 0V and 12V.
```

Comment:
Is it worth specifying this value can be negative?

Reply:
Yea, the range should be -12 to 12.